docs/plugin_development.rst

```diff
@@ -352,48 +427,57 @@ After ``cookicutter`` finishes its job, you'll get the following tree structure
 
 In the ``spyder`` directory you'll find the following files:
 
-* ``api.py``: where the functionality of the plugin is exposed to the rest of Spyder. That would allow additional functionality to be added from other plugins.
+* ``api.py``.
+  Exposes the functionality of the plugin to the rest of Spyder.
+  That allows adding additional functionality from other plugins.
 
-* ``plugin.py``: is the core of the plugin. Depending on the type of plugin we created, here you'll see an instance of ``SpyderDockablePlugin`` or ``SpyderPluginV2``.
+* ``plugin.py``.
+  This is the core of the plugin.
+  Depending on the type of plugin created, you will see an instance of either ``SpyderDockablePlugin`` or ``SpyderPluginV2``.
 
   * If it is a ``SpyderPluginV2`` you should set a constant class named ``CONTAINER_CLASS`` with an instance of ``PluginMainContainer``.
   * If it is a ``SpyderDockablePlugin`` you should set a constant class named ``WIDGET_CLASS`` with an instance of ``PluginMainWidget``.
 
-* ``container.py``: only used for ``SpyderPluginV2`` plugins. This file contains an instance of ``PluginMainContainer`` that holds a reference to all graphical elements (or widgets) that the plugin is going to add to the interface. This is necessary because Qt requires widgets to be children of other widgets before using them (otherwise they appear as floating windows). Since ``SpyderPluginV2`` is not a widget, we need a data structure (i.e. the container) that is a widget for that.
+* ``container.py``.
+  It is only used for ``SpyderPluginV2`` plugins.
+  This file contains an instance of ``PluginMainContainer``, which contains a reference to all graphical elements (or widgets) that the plugin will add to the interface.
+  This is necessary because Qt requires widgets to be children of other widgets before using them (otherwise they appear as floating windows).
+  Since ``SpyderPluginV2`` is not a widget, we need a data structure (i.e. the container) that is a widget for it.
 
-* ``widgets.py``: in this file we will add the graphical components of our plugin. If it is of type ``SpyderPluginV2`` and it does not have widgets, then it is not necessary.
-  We can also place here the instance of ``PluginMainWidget`` necessary for ``SpyderDockablePlugin``, if we are developing that kind of plugin.
+* ``widgets.py``.
+  Contains the graphical components of our plugin.
+  If the plugin is of type ``SpyderPluginV2`` and it does not have widgets, then it is not needed.
+  If the plugin is of type ``SpyderDockablePlugin``, this is where we can place the instance of ``PluginMainWidget`` that is needed for it.
 
-* ``confpage.py``: this is where you specify the configuration page that will be displayed in ``Preferences``, so that the user can adjust the options of our plugin.
+* ``confpage.py``.
+  It includes the specific configuration page that will be displayed in :guilabel:`Preferences`, so that the user can adjust the options of our plugin.
```

Comment:
```suggestion
  Includes the configuration page that will be displayed in :guilabel:`Preferences`, so that the user can adjust the options of our plugin.
```

Reply:
Why not start this sentence with `it`?

Reply:
To match the previous ("Contains the...", etc.) and the general form of this type of list